Assets/Scripts/ActionMove.cs

```diff
@@ -60,9 +64,14 @@ protected void Move()
                 spentActionPoints += tile.GetMoveCost();
                 path.Pop();
             }
+            if (path.Count > reserve_tiles)
+            {
+                anim.SetBool("IsWalking", true);
```

Comment:
Does this need to be in another if block?

Reply:
I find it looks better if it does - putting it in a block like that prevents the character from sometimes "moonwalking" an extra step animation after it reaches the center of the tile and is done moving.